docs/integrations/global-intelligence/kubernetes-devops.md

```diff
@@ -0,0 +1,180 @@
+
+### Dashboard Filters  
+
+**Each dashboard has a set of filters** that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard. 
+
+:::note
+You can use filters to drill down and examine the data on a granular level. 
+:::
+
+![img](/img/kubernetes/kubernetes-dashboard-filter.png)
+
+Select a **cluster**, **namespace**, and **deployment**. The **history_percent** represents the percent of 15m epocks over the prior 8 days over which Kubernetes metrics data was available for analysis for a given deployment-container combination. Percentages less than 100% represent deployment-container combinations that were new, transient or otherwise inactive for some portion of the prior 8 days.
+
+:::important
+The dashboard time range is used to fetch deployment-container combinations that have been active with Time Series data in the time range. Increasing the time range increases the number of such deployment-container combinations. To focus the optimization on the most active containers, start with a time range of 15m. For additional containers, gradually expand the time range.
+
+The `infer` operator is the interface to a machine learning model. It is intended for use only in this application.
+
+The infrastructure tab of the [Entity Inspector](../../dashboards-new/drill-down-to-discover-root-causes.md) is not compatible with Global Intelligence for Kubernetes DevOps and renders a blank page on Dashboards Summary for Kubernetes Deployment and Detail for Kubernetes Deployment. This is because Global Intelligence apps analyze anonymized customer data and do not have a way to identify customer entities required for Entity Inspector. 
+
+Deployment-container combinations that have no requests or limits set are not candidates for optimization. No recommendations are computed in these cases. 
+:::
+
+### Concepts, Definitions and Methodology
+
+It is relatively well known that DevOps staff concerned about Out of Memory (OOM) and CPU throttling may set high memory and CPU requests on their containers. This results in situations where Kubernetes clusters are over-provisioned compared to actual usage. Conversely, Kubernetes clusters may be underprovisioned, leading to risk of OOM and throttling and resulting downtime. 
+
+The slack is the difference between usage and requests measures cost for both cpu (cores) and memory (GB). Higher slack implies higher costs as cpu and memory used are less than what has been provisioned for a given container. Conversely negative slack, while saving costs, increase risk for container eviction due to memory overages or container throttling. These latter events impact application availability and performance and may lead to incidents. The cost-risk tradeoff is diagrammed below. 
```

Comment:
```suggestion
The slack is the difference between usage and requests measures cost for both cpu (cores) and memory (GB). Higher slack implies higher costs as cpu and memory used are less than what has been provisioned for a given container. Conversely, negative slack, while saving costs, increased the risk of container eviction due to memory overages or container throttling. These latter events impact application availability and performance and may lead to incidents. The cost-risk tradeoff is diagrammed below. 
```

Reply:
@kimsauce The first sentence (starting with "The slack is") doesn't make sense to me, in particular "requests measures cost". Maybe there's an extra or missing word?